spython/main/parse/docker.py

```diff
@@ -316,6 +319,25 @@ def _expose(self, line):
         return self._comment("# %s" %line)
 
 
+    def _user(self, line):
```

Comment:
Just for clarity I'd suggest this wording:

```
The user directive. Since Singularity users are the same inside and outside
the container at runtime, most common uses for the ``USER`` directive are
unsupported. Build-time actions in Singularity *are* performed by ``root``,
and equivalent functionality could be achieved with 
``RUN su <username> -c "your command"``.
```